doc/modules/cross_validation.rst

```diff
@@ -554,3 +554,17 @@ Cross validation and model selection
 Cross validation iterators can also be used to directly perform model
 selection using Grid Search for the optimal hyperparameters of the
 model. This is the topic of the next section: :ref:`grid_search`.
+
+Cross validation of time series data
+====================================
+
+Time series data is characterised by the correlation between observations 
+that are near in time (autocorrelation), so it is very important to 
+evaluate our models on the "future" observations least like those that 
+are seen by our model before. However, the "classical" k-times 
```

Comment:
A comment I made seems to have disappeared. I suggested "Classical cross-validation techniques assume the samples are independent and identically distributed, and would result in unreasonable correlation between training and testing instances (yielding poor estimates of generalisation error) on time-series data." Again, I think this belongs before "so ..."